python/ray/serve/handle.py

```diff
@@ -201,6 +216,9 @@ def __reduce__(self):
     def __getattr__(self, name):
         return self.options(method_name=name)
 
+    def __del__(self):
+        self.stop_metrics_pusher()
```

Comment:
Can this throw an error? And in that case, what happens?

Reply:
The stop_metrics_pusher is not supposed to throw errors. GC.should call _del_ func successfully.

Reply:
Got it, that sounds good.